crates/bevy_gltf/src/loader/morph.rs

```diff
@@ -0,0 +1,85 @@
+use std::iter;
+
+use bevy_render::mesh::morph::{MorphAttributes, VisitAttributes, VisitMorphTargets};
+use gltf::{
+    accessor::Iter,
+    mesh::{util::ReadMorphTargets, Reader},
+    Buffer,
+};
+
+pub(super) struct PrimitiveMorphTargetAttributes<'s> {
```

Comment:
what does the `'s` stands for here?

Reply:
There was probably a reason at one point, but I should probably switch to `'a`.